shooter_hot_reload/main/enemycontroller.script

```diff
@@ -133,7 +133,7 @@ end
 function on_message(self, message_id, message, sender)
 	if message_id == hash("start") then
 		self.waiting = true
-		self.t = 0
+		self.t = 8.2
```

Comment:
Why?

Reply:
during testing the value was made higher to spawn enemy waves faster. I should have reverted the change afterward.